pkg/bindings/images/images.go

```diff
@@ -0,0 +1,189 @@
+package images
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/containers/libpod/pkg/api/handlers"
+	"github.com/containers/libpod/pkg/bindings"
+	"github.com/containers/libpod/pkg/inspect"
+)
+
+// Exists a lightweight way to determine if an image exists in local storage.  It returns a
+// boolean response.
+func Exists(ctx context.Context, nameOrID string) (bool, error) {
+	conn, err := bindings.GetConnectionFromContext(ctx)
+	if err != nil {
+		return false, err
+	}
+	response, err := conn.NewRequest(http.MethodGet, fmt.Sprintf("/images/%s/exists", nameOrID), nil, nil)
+	if err != nil {
+		return false, err
+	}
+	if response.StatusCode == http.StatusOK {
+		return true, nil
+	}
+	return false, nil
+}
+
+// List returns a list of images in local storage.  The all boolean and filters parameters are optional
+// ways to alter the image query.
+func List(ctx context.Context, all *bool, filters map[string][]string) ([]handlers.ImageSummary, error) {
```

Comment:
This should be []*handlers.ImageSummary to be consistent with other API endpoints